README.md

```diff
@@ -0,0 +1,21 @@
+# Core Snap
+
+This repository contains the official Core snap. The core snap plays
+
+## Reporting Issues
+
+Please report all issues on the Launchpad project page
+https://bugs.launchpad.net/snapd/+filebug
+
+## Building
+
+To build the gadget snap locally please use `snapcraft`.
```

Comment:
i guess you want "core" here ... and since snapcraft will pull in an LP build system here you need sudo for it.